src/selection.ts

```diff
@@ -0,0 +1,62 @@
+export enum SelectionTypes {
```

Comment:
Good to know, thanks! I've been mainly using these enums as a way to do both the type checking but also to avoid hardcoded constants. I think I'd need to emulate `ScaleType` in that regard, then. For my own edification, could you explain why we've moved away from the enums? Thanks!